RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronID_cff.py

```diff
@@ -11,10 +11,20 @@
     )
 
 from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
-run2_miniAOD_UL.toModify(
+from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
+from Configuration.Eras.Modifier_bParking_cff import bParking
+( run2_miniAOD_UL | run2_miniAOD_devel | bParking ).toModify(
     lowPtGsfElectronID,
     rho = "fixedGridRhoFastjetAll",
     ModelWeights = ["RecoEgamma/ElectronIdentification/data/LowPtElectrons/LowPtElectrons_ID_2020Sept15.root"],
     ModelThresholds = [-99.],
     Version = "V1",
 )
+run2_miniAOD_devel.toModify(
+    lowPtGsfElectronID,
+    ModelWeights = ["RecoEgamma/ElectronIdentification/data/LowPtElectrons/LowPtElectrons_ID_2020Nov28.root"],
```

Comment:
shouldn't devel use `LowPtElectrons_ID_2021May17` ?

Reply:
- (~bParking & run2_miniAOD_devel) uses 2020Nov28
- It overrides (~bParking & run2_miniAOD_UL), which uses 2020Sept15 (no-change policy)
- Only (bParking & run2_miniAOD_UL) uses 2021May17

Reply:
> (~bParking & run2_miniAOD_devel) uses 2020Nov28

let me rephrase my earlier question: why can't `(~bParking & run2_miniAOD_devel)` use `LowPtElectrons_ID_2021May17`?

Reply:
- The 2021May17 and 2020Nov28 models are trained with UL data sets.
- The 2021May17 model is trained with the priors pT > 1 GeV and the Very Loose working point for the ElectronSeed.
- The 2020Nov28 model is trained with the priors pT > 1 GeV and the Tight working point for the ElectronSeed.
- The bParking modifier determines whether we use the VL or Tight working point.